### PR TITLE
Fix/Add `programs` folder to source tarball

### LINKED ISSUE #56: Release tar ball missing programs
**Describe the bug**
The programs directory is missing in the tar.xz release tar ball.
( `vsUTCS_v0.7.0-Source.tar.xz` )

### PR DESCRIPTION
**Pull request by stephengtuggy**
Thank you for submitting a pull request and becoming a contributor to Vega Strike: Upon the Coldest Sea.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [ ] This is a documentation change only

Issues:
- Please list any related issues
- Addresses #56 for 0.7.x

Purpose:
- What is this pull request trying to do? Fix the bug where the `programs` directory is not included in the Source tarballs
- What release is this for? 0.7.1
- Is there a project or milestone we should apply this to?

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -24,68 +24,72 @@
...
+    \\\\.git/
+    \\\\.vs/
+    \\\\.vscode/
+    ".*\\\\.pyc"
+)
+
+INSTALL(DIRECTORY .vegastrike/ DESTINATION share/vegastrike/.vegastrike)
 
 INSTALL(DIRECTORY ai/ DESTINATION share/vegastrike/ai
         PATTERN "*.am" EXCLUDE)
```
Comment (BenjamenMeyer) [inline CMakeLists.txt:47]: your able to remove the `PATTERN "*.am" EXCLUDE` portion from a few lines, could we extend that to all of the lines with this change?

**Review by BenjamenMeyer — APPROVED** [review]
One non-blocking comment; LGTM - would be nice to extend one of the changes to everything else too.

File: CMakeLists.txt (hunk 1 above)
Comment (stephengtuggy) [reply CMakeLists.txt:47]: @BenjamenMeyer No, unfortunately. I tried it, and it didn't work without `PATTERN "*.am" EXCLUDE` on at least a few lines. It brought in Makefile.am files, which evidently should be excluded.

Comment (BenjamenMeyer) [reply CMakeLists.txt:47]: @stephengtuggy thanks!

**Comment by BenjamenMeyer** [comment]
@stephengtuggy please port this to master

**Pull request merged**